Mathlib/Init.lean

```diff
@@ -5,6 +5,10 @@ import Mathlib.Tactic.Linter.DocString
 import Mathlib.Tactic.Linter.GlobalAttributeIn
 import Mathlib.Tactic.Linter.HashCommandLinter
 import Mathlib.Tactic.Linter.Header
+-- This linter is disabled by default, but downstream projects may want to enable it: to enable
+-- this, we import the linter here. Otherwise, downstream projects would need to import the linter
+-- in every file, to ensure it is available.
```

Comment:
```suggestion
```
Optional: I personally think this comment isn't even necessary, though I don't mind if you keep it

Reply:
I have shortened it a bit.